src/routes/(app)/checkout/+page.server.ts

```diff
@@ -414,8 +416,9 @@ export const actions = {
 					}
 				},
 				cart,
-				shippingAddress: shippingInfo?.shipping,
-				billingAddress: billingInfo?.billing || shippingInfo?.shipping,
+				shippingAddress: shippingInfo?.shipping as OrderAddress,
+				billingAddress:
+					(billingInfo?.billing as OrderAddress) || (shippingInfo?.shipping as OrderAddress),
```

Comment:
Why this change?

Reply:
It's show an error in the lint and in the code

Reply:
It's because the zod types are messed up for `shippingInfo`.

For example you can add the `phone` optional field for POS users too, it should fix it. Another solution is to refactor: `firstName: isPos ? z.string().default('') : z.string().min(1), ...`.

In your editor you can hover over variables to see the types. You can see that the type for `shippingInfo` is wrong.

Please fix problems at the source instead of casting to hide the issues. Here the editor warns you that you didn't handle the POS case. (eg POS users's phone numbers are ignored)